### PR TITLE
Update reportengine.php

### PR DESCRIPTION
**Pull request by magicsunday**
Handle given precision parameter in reports engine xml files.

### CODE REVIEW

File: reportengine.php [hunk 1]
```diff
@@ -215,8 +215,8 @@
 			foreach ($options as $option) {
 				$opt                   = explode('=>', $option);
 				list($value, $display) = $opt;
-				if (preg_match('/^I18N::number\((.+)\)$/', $display, $match)) {
-					$display = I18N::number($match[1]);
+                		if (preg_match('/^I18N::number\((.+?)(,([\d+]))?\)$/', $display, $match)) {
+                    			$display = I18N::number($match[1], isset($match[3]) ? $match[3] : 0);
```
Comment (fisharebest) [inline reportengine.php]: Can you convert spaces to tabs for indentation.
http://wiki.webtrees.net/en/Coding_Standards

Maybe make the second pattern non-capturing.  e.g. `(,` becomes `(?:,` - or maybe even `(?:, *`

**Comment by Amgine0** [comment]
Nice! but are not differences to the standard normally stated as EXCEPT?

File: reportengine.php (hunk 1 above)
Comment (magicsunday) [inline reportengine.php]: uh.. I did the change just here in github.

**Comment by magicsunday** [comment]
Please take another look at this. I changed the spaces to tabs.

**Pull request merged**